RollbarReactNative.podspec

```diff
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.license      = 'MIT'
   s.author       = { 'Rollbar' => 'support@rollbar.com' }
   s.platform     = :ios, '13.0'
-  s.source       = { :git => 'https://github.com/rollbar/rollbar-react-native.git', :tag => '1.0.0-beta.4' }
+  s.source       = { :git => 'https://github.com/rollbar/rollbar-react-native.git', :tag => '1.0.0-beta.5' }
```

Comment:
Your tags start with `v`. This tag definition is not correct. This causes the following error while trying to build the pods:

```
⚠️  Something went wrong running `pod install` in the `ios` directory.
Command `pod install` failed.
└─ Cause: Error installing RollbarReactNative
[!] /usr/bin/git clone https://github.com/rollbar/rollbar-react-native.git /var/folders/cl/8w0vfzkn113696rlz3sglw3h0000gn/T/d20241114-2312-sn9h1v --template= --single-branch --depth 1 --branch 1.0.0-beta.5

Cloning into '/var/folders/cl/8w0vfzkn113696rlz3sglw3h0000gn/T/d20241114-2312-sn9h1v'...
warning: Could not find remote branch 1.0.0-beta.5 to clone.
fatal: Remote branch 1.0.0-beta.5 not found in upstream origin
```

When I manually put `v` prefix, it works as expected.